doc/source/train/examples/lightning/lightning_cola_advanced.ipynb

```diff
@@ -40,11 +40,43 @@
     "SMOKE_TEST = True"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Uncomment and run the following line in order to install all the necessary dependencies:"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [],
+   "source": [
+    "# !pip install numpy datasets \"transformers>=4.19.1\" \"pytorch_lightning>=1.6.5\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's start by importing the needed libraries:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/ray/anaconda3/lib/python3.9/site-packages/xgboost/compat.py:31: FutureWarning: pandas.Int64Index is deprecated and will be removed from pandas in a future version. Use pandas.Index with the appropriate dtype instead.\n",
+      "  from pandas import MultiIndex, Int64Index\n"
+     ]
+    }
+   ],
    "source": [
```

Comment:
I think we can exclude these types of outputs that aren't valuable for the user.

Reply:
Done